spec/requests/documentation/documents_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Documentation
+  RSpec.describe 'Documents', type: :request do
+    include Engine.routes.url_helpers
+
+    fixtures 'documentation/pages'
+
+    let(:page) { documentation_pages(:comedor_home_page) }
+    let(:document_file) { fixture_file_upload('file.pdf', 'application/pdf') }
+
+    describe 'GET /index' do
```

Comment:
I suggest showing the completed path

Reply:
I just do it and it looks really ugly, I really prefer the action instead the full path, is really hard to read.